phenol-annotations/src/test/java/org/monarchinitiative/phenol/annotations/obo/hpo/HpoDiseaseAnnotationParserTest.java

```diff
@@ -25,23 +27,22 @@
  * These tests were constructed with a mini hpo file and a mini annotation file -- look at them to
  * figure out the logic!
  */
+@Disabled("Disabled until we have a toy hpo.json")
 class HpoDiseaseAnnotationParserTest {
 
   private static Map<TermId, HpoDisease> diseaseMap;
 
   @BeforeAll
-  public static void init() throws IOException {
+  public static void init() throws Exception {
     final String hpOboPath = "/hp_head.obo";
     URL hpOboURL = HpoCategoryMapTest.class.getResource(hpOboPath);
     if (hpOboURL == null) {
       throw new IOException("Could not find hpOboPath at " + hpOboPath);
     }
     File file = new File(hpOboURL.getFile());
     Ontology hpoOntology = OntologyLoader.loadOntology(file);
-    String phenoAnnot = "/annotations/phenotype_hpoa_head.tab";
-    URL annotURL = HpoCategoryMapTest.class.getResource(phenoAnnot);
-    String annotationPath = annotURL.getFile();
-    diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(annotationPath, hpoOntology);
+    URL annotURL = HpoCategoryMapTest.class.getResource("/annotations/phenotype_hpoa_head.tab");
+    diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(Paths.get(annotURL.toURI()), hpoOntology);
```

Comment:
Does this work on the module path?

```java  
  URL annotURL = HpoCategoryMapTest.class.getResource("/annotations/phenotype_hpoa_head.tab");
  diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(Paths.get(annotURL.toURI()), hpoOntology);
```
Wouldn't this be simpler:
```java  
  Path annotPath = Path.of("src/test/resources/annotations/phenotype_hpoa_head.tab");
  diseaseMap = HpoDiseaseAnnotationParser.loadDiseaseMap(annotPath, hpoOntology);
```

Reply:
It is true that your suggestion would be simpler in this case. However, my preference is to keep test files as close to the test classes as possible. This means that I do not store the test files in `/annotations/phenotype_hpoa_head.tab`, but at `src/test/resources/org/monarchinitiative/phenol/annotations/obo/hpo`. Then I get the path by using `HpoDiseaseAnnotationParserTest.class.getResource('phenotype_hpoa_head.tab')`. This approach simplifies management of the test files; if the test is removed, the test files are removed too. This is obviously at the cost of the possible test file duplication.

The resource loading works on module path, at least for loading test files.